src/components/cards/HeaderCArd.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+interface Props {
+  title: string;
+  description: string;
+}
+
+const HeaderCArd = ({ title, description }: Props) => {
```

Comment:
Can you name the name of the component in proper `PascalCase`

Reply:
The file name should be `Header`. since the file is within the `cards` directory. There is no need to include the `Card` suffix into the file name

Reply:
It is fixed